Dockerfile

```diff
@@ -1,5 +1,22 @@
-FROM ubuntu:20.04
+# build buck from source
+FROM ubuntu:20.04 AS buck
+
+ARG BUCK_VERSION=2021.01.12.01
+ENV ANT_OPTS="-Xmx4096m"
+RUN apt update  && apt install  -y --no-install-recommends \
+    ant \
+    git \
+    openjdk-11-jdk-headless \
+    python3 \
+    python-is-python3
+# install buck by compiling it from source. We also remove the buck repo once it's built.
+RUN git clone --depth 1 --branch v${BUCK_VERSION} https://github.com/facebook/buck.git \
+    && cd buck \
+    && ant \
+    && ./bin/buck build buck --out /tmp/buck.pex
```

Comment:
I believe you need a `--config java.target_level=11 --config java.source_level=11` here

Reply:
thanks for the comments. I'll try to build it on my local machine, hope everything works